lib/sift/filter.rb

```diff
@@ -21,10 +24,8 @@ def validation(_sort)
     def apply!(collection, value:, active_sorts_hash:, params: {})
       if not_processable?(value)
         collection
-      elsif should_apply_default?(value)
-        default.call(collection)
       else
-        handler.call(collection, parameterize(value), params, scope_params)
+        handler(value, params).call(collection)
```

Comment:
A fairly large change here: handlers are initialized with data from the request, but they are always called with just the collection. This allows us to treat the default handler, where handler, and scope handler as having the same interface once they are constructed.